packages/csvviewer-extension/src/searchprovider.ts

```diff
@@ -72,11 +72,29 @@ export class CSVSearchProvider implements ISearchProvider {
    *
    * @returns A promise that resolves once the action has completed.
    */
-  highlightPrevious(): Promise<ISearchMatch | undefined> {
+  async highlightPrevious(): Promise<ISearchMatch | undefined> {
     this._target.content.searchService.find(this._query, true);
     return undefined;
   }
 
+  /**
+   * Replace the currently selected match with the provided text
+   * Not implemented in the CSV viewer as it is read-only.
+   * @returns A promise that resolves once the action has completed.
```

Comment:
I think perhaps an empty line is needed before the @returns here and in the following function?

Reply:
Resolved in the latest commit.